lisa/operating_system.py

```diff
@@ -179,7 +201,23 @@ def install_packages(
         if self._first_time_installation:
             self._first_time_installation = False
             self._initialize_package_installation()
-        self._install_packages(package_names)
+
+        return self._install_packages(package_names, signed)
+
+    # TODO: Immplement update_packages
+    def update_packages(
+        self, packages: Union[str, Tool, Type[Tool], List[Union[str, Tool, Type[Tool]]]]
+    ) -> None:
+        pass
```

Comment:
raise not implemented error, if it's TODO. It avoid others call the method, but it doesn't affect actually.

Reply:
> raise not implemented error, if it's TODO. It avoid others call the method, but it doesn't affect actually.

I am going to add to it in this PR itself...in next iteration